### ISSUE TITLE
system_keyspace - update compaction history failed, followed by segmentation fault on a different shard

### ISSUE THREAD

**Issue by k0machi** [scylladb/scylla#9924]
*Installation details*
Kernel version: `5.4.0-1035-aws`
Scylla version (or git commit hash): `4.6.rc2-0.20220102.e8a1cfb6f with build-id 5d7b96e39c909424e8224207a162fc2c82b67214`
Cluster size: 6 nodes (i3.8xlarge)
Scylla running with shards number (live nodes):
   longevity-1000-keyspaces-4-6-db-node-01062ed8-1 (34.242.213.235 | 10.0.3.150): 30 shards
   longevity-1000-keyspaces-4-6-db-node-01062ed8-2 (34.242.40.208 | 10.0.1.252): 30 shards
   longevity-1000-keyspaces-4-6-db-node-01062ed8-3 (54.73.70.211 | 10.0.1.63): 30 shards
   longevity-1000-keyspaces-4-6-db-node-01062ed8-4 (54.76.27.84 | 10.0.0.84): 30 shards
   longevity-1000-keyspaces-4-6-db-node-01062ed8-5 (54.73.100.154 | 10.0.0.216): 30 shards
   longevity-1000-keyspaces-4-6-db-node-01062ed8-6 (52.214.218.88 | 10.0.0.170): 30 shards
OS (RHEL/CentOS/Ubuntu/AWS AMI): `ami-07d92096e7ab05aae` (aws: eu-west-1)

Test: `longevity-multi-keyspaces-60h-test`
Test name: `longevity_test.LongevityTest.test_batch_custom_time`
Test config file(s):
- [longevity-multi-keyspaces.yaml](https://github.com/scylladb/scylla-cluster-tests/blob/2c3e4506cee8b75c0f4e89b0aefb9f335c52d6dd/test-cases/longevity/longevity-multi-keyspaces.yaml)

**Issue description**

Nothing too concrete, this happened during normal stress command run with no nemesis running.

```
2022-01-15 17:20:06.486 <2022-01-15 17:19:50.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2838162 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
2022-01-15T17:19:50+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 7] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:20:07.145 <2022-01-15 17:20:00.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2838305 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
2022-01-15T17:20:00+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 15] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:20:39.433 <2022-01-15 17:20:39.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2838768 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
2022-01-15T17:20:39+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 2] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:20:40.874 <2022-01-15 17:20:40.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2838795 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
2022-01-15T17:20:40+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 5] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:20:40.875 <2022-01-15 17:20:40.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2838796 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
2022-01-15T17:20:40+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 9] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:20:47.682 <2022-01-15 17:20:47.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2838910 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
2022-01-15T17:20:47+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 10] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:21:38.314: (PrometheusAlertManagerEvent Severity.NORMAL) period_type=begin event_id=fb3d1a24-11a3-4e52-915c-bce5e8b465fe: alert_name=nodeLocalErrors node=10.0.0.216 start=2022-01-15T17:21:04.791Z end=2022-01-15T17:25:04.791Z description=Some operation failed at the replica side updated=2022-01-15T17:21:37.990Z state=active fingerprint=45e5fac76ace7210 labels={'advisor': 'operationError', 'alertname': 'nodeLocalErrors', 'dashboard': 'scylla-detailed', 'instance': '10.0.0.216', 'monitor': 'scylla-monitor', 'severity': '1'}
2022-01-15 17:22:46.415 <2022-01-15 17:22:46.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2840524 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
2022-01-15T17:22:46+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 3] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:22:50.313 <22-01-15 17:22:50.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2840577 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
022-01-15T17:22:50+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 28] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:22:50.314 <2022-01-15 17:22:50.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2840578 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
2022-01-15T17:22:50+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 17] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:24:52.722 <2022-01-15 17:23:55.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2841365 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
2022-01-15T17:23:55+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 1] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:26:35.368 <2022-01-15 17:25:53.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2842746 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
2022-01-15T17:25:53+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 9] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:26:35.589 <2022-01-15 17:26:01.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2842826 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
2022-01-15T17:26:01+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 23] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:26:35.871 <2022-01-15 17:26:11.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2842927 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
2022-01-15T17:26:11+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 27] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:26:36.051 <2022-01-15 17:26:17.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2842989 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
2022-01-15T17:26:17+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 25] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:26:36.055 <2022-01-15 17:26:17.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2842990 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
2022-01-15T17:26:17+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 3] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:26:36.327 <2022-01-15 17:26:27.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2843085 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
2022-01-15T17:26:27+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 4] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:26:52.320 <22-01-15 17:26:51.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2843382 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
022-01-15T17:26:51+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 9] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:27:34.703 <22-01-15 17:27:34.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2843874 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
022-01-15T17:27:34+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 11] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:27:35.408 <2022-01-15 17:27:34.000>: (DatabaseLogEvent Severity.WARNING) period_type=one-time event_id=aeb52def-5af0-4716-8256-5d8524a92cf2: type=SYSTEM_PAXOS_TIMEOUT regex=(mutation_write_|Operation timed out for system.paxos|Operation failed for system.paxos) line_number=2843881 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
2022-01-15T17:27:34+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !     ERR |  [shard 26] system_keyspace - update compaction history failed: exceptions::mutation_write_timeout_exception (Operation timed out for system.compaction_history - received only 0 responses from 1 CL=ONE.): ignored
2022-01-15 17:27:44.384 <2022-01-15 17:27:44.000>: (DatabaseLogEvent Severity.ERROR) period_type=one-time event_id=464ba803-3013-4d2e-9291-d6d0788c8534: type=SEGMENTATION regex=segmentation line_number=2843990 node=longevity-1000-keyspaces-4-6-db-node-01062ed8-5
2022-01-15T17:27:44+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !    INFO | Segmentation fault on shard 0.
2022-01-15T17:27:44+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !    INFO | Backtrace:
2022-01-15T17:27:44+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !    INFO |   0x3db44f8
2022-01-15T17:27:44+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !    INFO |   0x3de52e6
2022-01-15T17:27:44+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !    INFO |   0x4433f91
2022-01-15T17:27:44+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !    INFO |   0x7f21fd9f5a1f
2022-01-15T17:27:44+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !    INFO |   0x26af7df
2022-01-15T17:27:44+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !    INFO |   0x3dc6ec4
2022-01-15T17:27:44+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !    INFO |   0x3dc82b7
2022-01-15T17:27:44+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !    INFO |   0x3dc74fc
2022-01-15T17:27:44+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !    INFO |   0x3d70d8e
2022-01-15T17:27:44+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !    INFO |   0x3d70106
2022-01-15T17:27:44+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !    INFO |   0xf406b5
2022-01-15T17:27:44+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !    INFO |   /opt/scylladb/libreloc/libc.so.6+0x27b74
2022-01-15T17:27:44+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !    INFO |   0xf3d86d
2022-01-15T17:27:44+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !    INFO | scylla[731206]: segfault at 138 ip 00000000026af7e0 sp 00007fffcdb42600 error 6 in scylla[f3d000+37f8000]
2022-01-15T17:27:44+00:00 longevity-1000-keyspaces-4-6-db-node-01062ed8-5 !    INFO | Code: 08 03 00 00 49 89 ae 10 03 00 00 4c 01 e2 49 89 96 18 03 00 00 49 8b 45 08 4c 8b 6c 24 18 48 8b 6c 24 10 48 8b 0d 98 3f 10 02 <48> 89 88 38 01 00 00 48 8b 43 28 48 85 c0 0f 84 81 00 00 00 48 83
```

after decoding backtrace we get:

```
[Backtrace #0]
void seastar::backtrace<seastar::backtrace_buffer::append_backtrace()::{lambda(seastar::frame)#1}>(seastar::backtrace_buffer::append_backtrace()::{lambda(seastar::frame)#1}&&) at ./build/release/seastar/./seastar/include/seastar/util/backtrace.hh:59
 (inlined by) seastar::backtrace_buffer::append_backtrace() at ./build/release/seastar/./seastar/src/core/reactor.cc:756
 (inlined by) seastar::print_with_backtrace(seastar::backtrace_buffer&, bool) at ./build/release/seastar/./seastar/src/core/reactor.cc:786
seastar::print_with_backtrace(char const*, bool) at ./build/release/seastar/./seastar/src/core/reactor.cc:798
 (inlined by) seastar::sigsegv_action() at ./build/release/seastar/./seastar/src/core/reactor.cc:3712
 (inlined by) operator() at ./build/release/seastar/./seastar/src/core/reactor.cc:3698
 (inlined by) __invoke at ./build/release/seastar/./seastar/src/core/reactor.cc:3694
wasmtime_runtime::traphandlers::unix::trap_handler at wasmtime_runtime.9xv9ixev-cgu.15:?
?? ??:0 <-- 0x7f21fd9f5a1f
std::__atomic_base<long>::load(std::memory_order) const at /usr/lib/gcc/x86_64-redhat-linux/11/../../../../include/c++/11/bits/atomic_base.h:481
 (inlined by) seastar::lowres_clock_impl::steady_now() at ././seastar/include/seastar/core/lowres_clock.hh:61
 (inlined by) seastar::lowres_clock::now() at ././seastar/include/seastar/core/lowres_clock.hh:119
 (inlined by) db::commitlog::segment::reset_sync_time() at ./db/commitlog/commitlog.cc:651
 (inlined by) db::commitlog::segment_manager::new_segment() at ./db/commitlog/commitlog.cc:1634
seastar::reactor::run_tasks(seastar::reactor::task_queue&) at ./build/release/seastar/./seastar/src/core/reactor.cc:2374
 (inlined by) seastar::reactor::run_some_tasks() at ./build/release/seastar/./seastar/src/core/reactor.cc:2783
seastar::reactor::do_run() at ./build/release/seastar/./seastar/src/core/reactor.cc:2952
seastar::reactor::run() at ./build/release/seastar/./seastar/src/core/reactor.cc:2835
seastar::app_template::run_deprecated(int, char**, std::function<void ()>&&) at ./build/release/seastar/./seastar/src/core/app-template.cc:228
seastar::app_template::run(int, char**, std::function<seastar::future<int> ()>&&) at ./build/release/seastar/./seastar/src/core/app-template.cc:128
main at ./main.cc:490
/opt/scylladb/libreloc/libc.so.6+0x27b74
0xf3d86d <-- __start
```

Restore Monitor Stack command: `$ hydra investigate show-monitor 01062ed8-bb91-4e08-91d5-b6a730ed0529`
Restore monitor on AWS instance using [Jenkins job](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id=01062ed8-bb91-4e08-91d5-b6a730ed0529)
Show all stored logs command: `$ hydra investigate show-logs 01062ed8-bb91-4e08-91d5-b6a730ed0529`

Test id: `01062ed8-bb91-4e08-91d5-b6a730ed0529`

Logs:
grafana - [https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220115_193403/grafana-screenshot-longevity-multi-keyspaces-60h-test-scylla-per-server-metrics-nemesis-20220115_193750-longevity-1000-keyspaces-4-6-monitor-node-01062ed8-1.png](https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220115_193403/grafana-screenshot-longevity-multi-keyspaces-60h-test-scylla-per-server-metrics-nemesis-20220115_193750-longevity-1000-keyspaces-4-6-monitor-node-01062ed8-1.png)
grafana - [https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220115_193403/grafana-screenshot-overview-20220115_193403-longevity-1000-keyspaces-4-6-monitor-node-01062ed8-1.png](https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220115_193403/grafana-screenshot-overview-20220115_193403-longevity-1000-keyspaces-4-6-monitor-node-01062ed8-1.png)
critical - [https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/critical-01062ed8.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/critical-01062ed8.log.tar.gz)
db-cluster - [https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/db-cluster-01062ed8.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/db-cluster-01062ed8.tar.gz)
debug - [https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/debug-01062ed8.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/debug-01062ed8.log.tar.gz)
email_data - [https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/email_data-01062ed8.json.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/email_data-01062ed8.json.tar.gz)
error - [https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/error-01062ed8.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/error-01062ed8.log.tar.gz)
event - [https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/events-01062ed8.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/events-01062ed8.log.tar.gz)
left_processes - [https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/left_processes-01062ed8.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/left_processes-01062ed8.log.tar.gz)
loader-set - [https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/loader-set-01062ed8.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/loader-set-01062ed8.tar.gz)
monitor-set - [https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/monitor-set-01062ed8.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/monitor-set-01062ed8.tar.gz)
normal - [https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/normal-01062ed8.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/normal-01062ed8.log.tar.gz)
output - [https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/output-01062ed8.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/output-01062ed8.log.tar.gz)
event - [https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/raw_events-01062ed8.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/raw_events-01062ed8.log.tar.gz)
sct - [https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/sct-01062ed8.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/sct-01062ed8.log.tar.gz)
summary - [https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/summary-01062ed8.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/summary-01062ed8.log.tar.gz)
warning - [https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/warning-01062ed8.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/01062ed8-bb91-4e08-91d5-b6a730ed0529/20220116_021831/warning-01062ed8.log.tar.gz)

[Jenkins job URL](https://jenkins.scylladb.com/job/scylla-4.6/job/longevity/job/longevity-multi-keyspaces-60h-test/4/)

**Comment by avikivity** [comment]
This is very odd. It crashed while loading a static variable.

**Comment by avikivity** [comment]
Where is the core dump?

**Comment by slivne** [comment]
I think this happend this may be similar to other cores we got (https://github.com/scylladb/scylla/issues/9896) - please note we are missing error handling fixes in commitlog 

we will spin 4.6.rc3 that has the fix for #9870 backported to 4.6 (7dc5abb) and retest

**Comment by slivne** [comment]
4.6.rc5 includes multiple new fixes (compared to all 4.6.rc4 and others) for commitlog the last is https://github.com/scylladb/scylla/commit/34d470967a05839b393be3915e41ba850038c1b0

there are no reports from QA on 4.6.rc5 that they have crashes related to commitlog closing this issue
